python/requirements_compiled.txt

```diff
@@ -522,31 +541,38 @@ typer==0.9.0
 types-pyyaml==6.0.12.2
 typing-extensions==4.5.0
 uc-micro-py==1.0.2
+ujson==5.8.0
 uri-template==1.3.0
 uritemplate==4.1.1
 urllib3==1.26.16
 uvicorn==0.22.0
 uvloop==0.17.0
 virtualenv==20.21.0
 wandb==0.13.4
+watchfiles==0.19.0
+wcmatch==8.4.1
 wcwidth==0.2.6
 webcolors==1.13
 webencodings==0.5.1
 websocket-client==1.6.1
 websockets==11.0.3
 werkzeug==2.1.2
-wheel==0.40.0
-widgetsnbextension==3.6.4
+wheel==0.41.0
+widgetsnbextension==3.6.5
 wrapt==1.15.0
 wurlitzer==3.0.3
 xgboost==1.7.6 ; python_version > "3.7"
 xlrd==2.0.1
 xmltodict==0.13.0
-xxhash==3.2.0
-y-py==0.5.9
+xxhash==3.3.0
```

Comment:
these `python_version > "3.7"` can be removed now?

Reply:
Auto-added by pip-compile, I believe it will disappear once we get rid of these markers in the input files